render/process.go

```diff
@@ -46,7 +46,7 @@ func (r processWithContainerNameRenderer) Render(rpt report.Report) Nodes {
 		if !ok {
 			continue
 		}
-		propagateLatests(container, p, docker.ContainerName)
+		p = propagateLatests(container, p, docker.ContainerName)
```

Comment:
That was a straight bug. How does it manifest?

Reply:
The absence of any ContainerName on Process nodes, I think.